elixir/apps/domain/lib/domain/accounts/account.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Domain.Accounts.Account do
   use Domain, :schema
 
+  @derive {Phoenix.Param, key: :slug}
```

Comment:
We should implement this protocol in web app, because in `domain` Phoenix is should not be added as a dependency:
```
defimpl Phoenix.Param, for: Domain.Accounts.Account do
  def to_param(account), do: account.slug || account.id
end
```

Reply:
Ah, ok, that makes sense. 👍   I'll get this updated

Reply:
Moved this to the `web` app as recommended